src/sgame/sg_main.cpp

```diff
@@ -2211,6 +2211,8 @@ void G_RunFrame( int levelTime )
 
 	G_CheckPmoveParamChanges();
 
+	int numBuildables[ BA_NUM_BUILDABLES ] = { 0 };
```

Comment:
Remove the 0, just `{}` (as is there should be a compile warning about too few elements to initialize)